consistent-sampling/src/main/java/io/opentelemetry/contrib/samplers/ConsistentAlwaysOffSampler.java

```diff
@@ -10,7 +10,9 @@
 @Immutable
 final class ConsistentAlwaysOffSampler extends ConsistentSampler {
 
-  private ConsistentAlwaysOffSampler() {}
+  private ConsistentAlwaysOffSampler() {
+    super(s -> RandomGenerator.getDefault().numberOfLeadingZerosOfRandomLong());
```

Comment:
@oertl what's the reason for having/using a ConsistentAlwaysOffSampler vs a normal always off sampler?

Reply:
The `ConsistentAlwaysOffSampler` sets an r-value when it is not present. This way, the spans of different child branches will all have the same r-value and the corresponding sampling decisions will be consistent. Furthermore, it sets the p-value to `OtelTraceState.getMaxP()` which corresponds to a probability of 0, so that a subsequent `ConsistentParentBasedSampler` can make the sampling decision based on the p-value and not on the sampled flag.